src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -380,6 +395,12 @@ public void Close()
             }
         }
 
+        /// <summary>
+        /// Open the window.
+        /// </summary>
+        /// <remarks>
+        /// This should be used if you want to setup your own loop; otherwise, you should use <see cref="WindowExtensions.Run"/> instead
+        /// </remarks>

```

Comment:
Was there not any documentation to inherit for this method? If not, these doc comments should be moved to the relevant interface in Windowing.Common instead